### ISSUE TITLE
Newest version is Messing with css of stacked elements.

### ISSUE THREAD

**Issue by Ceerbeerus** [ofekashery/vertical-stack-in-card#71]
The latest version of this card is messing with the css of elements below it.

Ex. button-Cards is no longer showing background-color but is transparent.
(This behaviour is new)

**Comment by sirotkaslo** [comment]
I'm having this issue too, will there be a fix for this? thank you!

**Comment by Mariusthvdb** [comment]
yep, just found out the same thing:

```
  - type: custom:vertical-stack-in-card
    cards:
      - type: picture
        image: /local/images/nmbs.png
        hold_action:
          action: url
          url_path: https://www.belgiantrain.be/nl
      - type: entities
        style: |
          ha-card {
            --ha-card-background: rgb(0,100,182);
            color: rgb(217,242,251);
          }
        title: Nmbs reizen
        show_header_toggle: false
        entities:
          - sensor.essen_brussel_noord
          - sensor.brussel_noord_essen

          - sensor.nmbs_live
          - sensor.nmbs_live_2
          - sensor.nmbs_live_3
          - sensor.nmbs_live_4
```

above style config does style the color: but the --ha-card-background won't work. For the sake of experiment I also tried background: and background-color:, and I've also tried a color name ('green') or hex value.

please have a look for a fix?
thanks

**Comment by josh929800** [comment]
Same issue for me...

**Comment by benct** [comment]
Users are reporting the same issue when placing [xiaomi-vacuum-card](https://github.com/benct/lovelace-xiaomi-vacuum-card/) in this card... ref. [this issue](https://github.com/benct/lovelace-xiaomi-vacuum-card/issues/28).

**Comment by jacekpaszkowski** [comment]
any chance to fix that issue?

**Comment by jacekpaszkowski** [comment]
I've checked and it works ok in v0.1.3. Issue appered in v0.2.0

**Comment by jacekpaszkowski** [comment]
just to note. i was able to workaround my issues usung card-mod repeting the styles which disappear after loading the card.

```yaml
[...]
    style: |
      ha-card {
        background-image: url('/local/images/xiaomi_vacuum_2.png')!important;
        background-size: cover!important;
        background-repeat: no-repeat!important;
        background-position: center center!important;
      }
    type: 'custom:xiaomi-vacuum-card'
```

**Comment by RomRider** [comment]
There is a feature for that in my new card which can replace vertical-stack-in-card
I've created a new card which is almost a drop-in replacement for vertical-stack-in-card and a bunch of other features:
https://community.home-assistant.io/t/stack-in-card-drop-in-replacement-for-vertical-stack-in-card/180072

**Comment by ofekashery** [comment]
Fixed in version [0.3.0](https://github.com/ofekashery/vertical-stack-in-card/releases/tag/v0.3.0), thanks!
